geode-core/src/main/java/org/apache/geode/management/internal/util/ManagementUtils.java

```diff
@@ -162,14 +163,20 @@ public static DistributedMember getDistributedMemberByNameOrId(String memberName
     Set<Region<?, ?>> rootRegions = cache.rootRegions();
 
     for (Region<?, ?> rootRegion : rootRegions) {
-      regionNames.add(rootRegion.getFullPath().substring(1));
+      try {
+        Set<Region<?, ?>> subRegions = rootRegion.subregions(true);
 
-      Set<Region<?, ?>> subRegions = rootRegion.subregions(true);
+        for (Region<?, ?> subRegion : subRegions) {
+          regionNames.add(subRegion.getFullPath().substring(1));
+        }
 
-      for (Region<?, ?> subRegion : subRegions) {
-        regionNames.add(subRegion.getFullPath().substring(1));
+      } catch (RegionDestroyedException ignored) {
```

Comment:
Are we sure this is the only exception this block of code can throw? If we want to prevent this block of code errors out, we should catch a general `Exception`, logs the exception, and then continue.

Reply:
It's not the only exception this block can throw. I like your suggestion of catching general `Exception`. Would it be OK to log the caught exception as "debug" level? (My main purpose with this change is to prevent RegionDestroyedException from showing up in the logs under normal conditions when it is thrown from this function.)

Reply:
would "info" level be ok? not a big stack trace, just enough information about why some region didn't show up in the list.

Reply:
We are performing a health check using the REST API /members endpoint while some regions are being created and destroyed and RegionDestroyedException gets logged quite often. We would like to suppress this exception because it's expected for the call to subregions() to throw this exception after a region is destroyed and it's confusing to users to see this exception in the logs. I am hesitant to log it as an info level exception because it will still show up in the logs under normal circumstances and will still be confusing to users.

Reply:
@jinmeiliao I changed this block of code to catch `Exception` instead of `RegionDestroyedException`, log a debug-level message, and then continue.